### PR TITLE
add inference case for bloom and update readme

### PR DESCRIPTION
**Pull request by sywangyi**
Signed-off-by: Wang, Yi A <yi.a.wang@intel.com>

## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Comment by sywangyi** [comment]
@hshen14 @yao-matrix

File: examples/optimization/pytorch/huggingface/language-modeling/inference/run_bloom.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+t_generate_start = time.time()
+generated = generate()
+t_generate_span = time.time() - t_generate_start
+for i, o, _ in generated:
+    print(f"{'-'*60}\nin={i}\nout={o}\n")
+
+### Benchmark
+
+if args.benchmark:
+    gc.collect()
```
Comment (hshen14) [inline examples/optimization/pytorch/huggingface/language-modeling/inference/run_bloom.py:104]: do we need to enable gc collect for benchmarking?

Comment (sywangyi) [reply examples/optimization/pytorch/huggingface/language-modeling/inference/run_bloom.py]: remove it

File: examples/optimization/pytorch/huggingface/language-modeling/inference/run_bloom.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
+import argparse
+import gc
```
Comment (hshen14) [inline examples/optimization/pytorch/huggingface/language-modeling/inference/run_bloom.py:2]: Please remove unused import

**Review by hshen14 — APPROVED** [review]
LGTM

**Pull request merged**